tracer/tracer.go

```diff
@@ -720,40 +703,35 @@ func (t *Tracer) insertKernelFrames(trace *host.Trace, ustackLen uint32,
 			ReturnAddress: true,
 		}
 
+		if !foundFileID {
+			kernelSymbolCacheMiss++
```

Comment:
Is this a cache miss? I think this an "unable to symbolize" event. Similar to below, if `t.kernelSymbols.LookupByAddress` below fails, there is no counter increase due to "no data available".